dev/com.ibm.websphere.org.eclipse.microprofile.faulttolerance.2.1/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+# IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.faulttolerance.2.1; singleton:=true
+
+Export-Package: \
+  org.eclipse.microprofile.faulttolerance;version=1.1, \
```

Comment:
I think this should be at least version 1.2

Reply:
Done, though should be reviewed when we update to the released version